packages/contracts-bedrock/scripts/deploy/VerifyOPCM.s.sol

```diff
@@ -338,6 +381,68 @@ contract VerifyOPCM is Script {
         return success;
     }
 
+    /// @notice Verifies that the immutable variables in the OPCM contract match expected values.
+    /// @param _opcm The OPCM contract to verify immutable variables for.
+    /// @return True if all immutable variables are verified, false otherwise.
+    function _verifyOpcmImmutableVariables(IOPContractsManager _opcm) internal returns (bool) {
+        console.log("  Verifying OPCM immutable variables...");
+
+        bool success = true;
+
+        // Get all OPCM getters and iterate over them
+        string[] memory allGetters = _getOpcmGetters();
+
+        for (uint256 i = 0; i < allGetters.length; i++) {
+            string memory functionName = allGetters[i];
+            string memory verificationMethod = expectedGetters[functionName];
+
+            // All getters must be accounted for in expectedGetters mapping
+            if (bytes(verificationMethod).length == 0) {
+                console.log("ERROR: Getter '%s' is not accounted for in expectedGetters mapping", functionName);
+                return false;
+            }
+
+            // Skip getters that don't need env var verification
+            if (keccak256(bytes(verificationMethod)) == keccak256(bytes("SKIP"))) {
+                continue;
+            }
+
+            // This getter should be verified via environment variable
+            string memory envVarName = verificationMethod;
+
+            // Get expected address from environment variable
+            // nosemgrep: sol-style-vm-env-only-in-config-sol
+            address expectedAddress = vm.envAddress(envVarName);
```

Comment:
You don't need `envVarName` here I don't think, you can simply use `vm.envAddress(verificationMethod)`